src/auth/negotiate/SSPI/negotiate_sspi_auth.cc

```diff
@@ -233,14 +233,17 @@ manage_request()
         c = (char *) SSP_ValidateNegotiateCredentials(decoded, decodedLen, &Done, &status, cred);
 
         if (status == SSP_ERROR) {
-            FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM |
-                          FORMAT_MESSAGE_IGNORE_INSERTS,
-                          nullptr,
-                          GetLastError(),
-                          MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),    /* Default language */
-                          (LPTSTR) & ErrorMessage,
-                          0,
-                          nullptr);
+            DWORD n = FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM |
```

Comment:
If possible, please use [AAA](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/) and/or `const` in new and changed code. For example:

```suggestion
            const auto n = FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM |
```

Similar approach has already been applied to FormatMessage() in WindowsErrorMessage().

Same for src/auth/ntlm/SSPI/ntlm_sspi_auth.cc, of course.

P.S. The above suggestion, if it compiles, may require reformatting of the affected code.